Documentation/dotnet-mage/README.md

```diff
@@ -56,3 +56,7 @@ Suppose that we have copied project output to a sub-folder `files` and our .NET
 * Create deployment manifest
 
 `mage.net -new Deployment -Install true -pub "My Publisher" -v 1.0.0.1 -AppManifest files\MyApp.manifest -t MyApp.application`
+
+### Update an existing application
+
+`mage.net -update MyApp.Application -v 1.0.0.2 -AppManifest files\MyApp.manifest -t MyApp.application`
```

Comment:
This line would update deployment manifest, but not the application manifest.

Here's what I propose for content of the new section "Update an existing application":

* Update application manifest

`mage.net -update files\MyApp.manifest -v 1.0.0.2

* Update deployment manifest

`mage.net -update MyApp.application -AppManifest files\MyApp.manifest -v 1.0.0.2

Reply:
Thanks Nikola, updated!

Reply:
> Thanks Nikola, updated!

Looks good - a small ask - in second command '-t' option is not needed during update, as the application manifest is already specified, so the command would look like:

`mage.net -update MyApp.Application -v 1.0.0.2 -AppManifest files\MyApp.manifest`

Reply:
@NikolaMilosavljevic Done! Thanks!